src/component/axis/AxisBuilder.ts

```diff
@@ -347,6 +348,16 @@ const builders: Record<'axisLine' | 'axisTickLabel' | 'axisName', AxisElementsBu
         fixMinMaxLabelShow(axisModel, labelEls, ticksEls);
 
         buildAxisMinorTicks(group, transformGroup, axisModel, opt.tickDirection);
+
+        const labelList = prepareLayoutList(labelEls.map(label => ({
+            label,
+            priority: label.style.text?.includes('primary') ? 1 : 0,
```

Comment:
@pissang I was not able to understand how to generate the priorities here so I did something very dirty to see if this works or not. Seems to be working but what is the proper way to assign the priority to the label? I think this should be done somewhere in https://github.com/apache/echarts/blob/master/src/scale/Time.ts as this method is used by many charts, if not all.

Reply:
`getTicks` method of `Time.ts` has a `level` property. It can be used as priority. And `labelEls` is created from the list of `getTicks`.